.github/workflows/ci.yaml

```diff
@@ -69,6 +74,22 @@ jobs:
           # as the placeholder for the auth token
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+      - name: Use snapshot version
```

Comment:
I would merge the steps `Use snapshot version` and `publish snapshot libraries` since there are correlated